specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2020-02-07-preview/hanaonazure.json

```diff
@@ -617,6 +617,10 @@
           "readOnly": true,
           "type": "string",
           "description": "The version of the payload running in the Collector VM"
+        },
+        "monitorSubnet": {
```

Comment:
>monitorSubnet [](start = 9, length = 13)

Is this property required?

Reply:
yes

Reply:
Then this is a breaking change that should be done in a new API version.

---
In reply to: [402059679](https://github.com/Azure/azure-rest-api-specs/pull/8915#discussion_r402059679) [](ancestors = 402059679)

Reply:
This is still a preview API version and the API is still in flux.
We also know that no one outside of our team is using this API yet.

Reply:
Got it. That's fine then. 

In general, I would recommend working in a development branch in the public or private Swagger repo when the API is still in flux. What's in `master` branch in this repo is considered public and available for use regardless of the `-preview` suffix on the API version.

---
In reply to: [402062621](https://github.com/Azure/azure-rest-api-specs/pull/8915#discussion_r402062621) [](ancestors = 402062621)

Reply:
I see, thanks!
Will do that in the future